src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSSource.java

```diff
@@ -84,7 +84,7 @@ public void enterTheGCSSourceMandatoryProperties() throws InterruptedException,
     CdfGcsActions.enterReferenceName();
     CdfGcsActions.enterProjectId();
     CdfGcsActions.getGcsBucket("gs://" + TestSetupHooks.gcsSourceBucketName + "/"
-                                 + PluginPropertyUtils.pluginProp("gcsCsvFile"));
+                                 + PluginPropertyUtils.pluginProp("gcsDataTypeTest2File"));
```

Comment:
This method `enterTheGCSSourceMandatoryProperties()`  is used in multiple scenarios. It will affect their execution. Do not change here.